main.nf

```diff
@@ -791,17 +793,24 @@ process transcript_to_gene {
     maxRetries 3
         
     input:
-        set val(espTag), val(expName), val(species), file(referenceFasta), file(referenceGtf) from REFS_FOR_T2GENE
+        set val(espTag), val(expName), val(species), file(referenceFasta), file(referenceGtf), val(isDroplet) from REFS_FOR_T2GENE_DROPLET.join(IS_DROPLET_FOR_T2G)
+        set val(espTag), val(expName), val(species), file(referenceCDNA) from REFS_FOR_T2GENE_NON_DROPLET
+       
 
     output:
         set val(espTag), file('transcript_to_gene.txt') into TRANSCRIPT_TO_GENE
 
     """
-    gtf2featureAnnotation.R --gtf-file $referenceGtf --version-transcripts \
-        --parse-cdnas $referenceFasta  --parse-cdna-field "transcript_id" --feature-type \
+    if [ $isDroplet == 'True' ]; then
+        pyroe make-splici ${referenceFasta} ${referenceGtf} 90 splici_transcriptome
```

Comment:
@pmb59 we also need the read length to build the splici transcriptome for the index, so I think we need to set the same read length for all experiments, because we only want to have one index per species. 
In the alevin-fry paper they write: "We also set the flank length as
the read length minus five, although the quantification results appear very robust to the specific flank length chosen (Supplementary Information)", so I think it should not be a problem to set one read length for all experiments. But we maybe should set it to 150.